personas/index.html

```diff
@@ -6,7 +6,7 @@
     <title>Tachyons Tutorial</title>
     <meta name="viewport" content="width=device-width, initial-scale=1">
     <link rel="stylesheet" href="https://unpkg.com/tachyons@4.8.1/css/tachyons.min.css">
-    <link rel="shortcut icon" type="image/png" href="http://www.dwyl.io/img/favicon.ico"/>
+    <link rel="shortcut icon" type="image/png" href="https://www.dwyl.io/img/favicon.ico"/>
```

Comment:
```suggestion
    <link rel="shortcut icon" type="image/png" href="https://www.dwyl.com/img/favicon.ico"/>
```
Sadly we no longer control the dwyl.io domain ... so needs to be .com 👌

Reply:
Thanks done